src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesEdit.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+import { Card, Checkbox, Col, Layout, Row, Select, TextArea, TextField } from '@folio/stripes/components';
+import { required } from '../../../util/validators';
+
+export default class ExternalDataSourcesEdit extends React.Component {
+  static propTypes = {
+    actionButtons: PropTypes.func,
+    input: PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      value: PropTypes.shape({
+        id: PropTypes.string,
+      }).isRequired,
+    }).isRequired,
+    meta: PropTypes.shape({
+      invalid: PropTypes.bool,
+      pristine: PropTypes.bool,
+      submitting: PropTypes.bool,
+    })
+  }
+
+  render() {
+    const { input: { value, name } } = this.props;
+    const kbadapterTypes = [{ value: 'org.olf.kb.adapters.EbscoKBAdapter', label: 'org.olf.kb.adapters.EbscoKBAdapter' }];
+    const recordTypes = [{ value: '1', label: 'Package' }];
+    return (
+      <Card
+        data-test-external-data-source-edit
+        headerStart={(
+          <strong>
+            {value.id ?
+              <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.editExternalKb" />
+              :
+              <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.newExternalKb" />}
+          </strong>
+        )}
+        headerEnd={this.props.actionButtons}
+      >
+        <Row>
+          <Col xs={4} md={4}>
+            <Field
+              component={TextField}
+              data-test-external-data-source-name-edit
+              label={<FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.name" />}
+              name={`${name}.name`}
+              required
+              validate={required}
+            />
+          </Col>
+          <Col xs={4} md={4}>
+            <Field
+              component={Select}
+              data-test-external-data-source-type-edit
+              dataOptions={kbadapterTypes}
+              label={<FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.type" />}
+              name={`${name}.type`}
+              required
+              validate={required}
+            />
+          </Col>
+          <Col xs={4} md={4}>
+            <Field
+              component={Select}
+              data-test-external-data-source-record-type-edit
+              dataOptions={recordTypes}
+              label={<FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.recordType" />}
+              name={`${name}.rectype`}
+              required
+              validate={required}
+            />
+          </Col>
+        </Row>
+        <Field
+          component={TextField}
+          label={<FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.uri" />}
+          name={`${name}.uri`}
+        />
+        <Layout className="padding-bottom-gutter">
```

Comment:
Is this because of specific `Checkbox` rendering? Curious...

Reply:
Yes, if not they look like this:

![image](https://user-images.githubusercontent.com/8953703/63607590-d9bd1e80-c5a0-11e9-8a36-09aa2becb6f9.png)